docs/SPEC.md

```diff
@@ -3172,23 +3172,22 @@ Given the following input table.
     
 ####  Moving Average
 
-Moving Average computes the moving averages of a series of records.
-It means the values of a user-defined period for a defined number of points,
+Moving Average computes the moving average the `_value` column.
```

Comment:
Nit: typo, I think you want to write "moving average **of** the `_value` column."

Reply:
Also, I'm curious why this was change to only work on the `_value` column, is it to be consistent with our other functions?

Reply:
Correct, this is to be consistent with `doubleEMA`, and `tripleEMA` since those have `_value` hard-coded into them.